src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+interface TodoFooterProps {
+  todos: Todo[];
+  selectedFilter: string;
+  setSelectedFilter: React.Dispatch<React.SetStateAction<string>>;
+  clearCompletedTodos: () => void;
+}
+
+const TodoFooter: React.FC<TodoFooterProps> = ({
+  todos,
+  selectedFilter,
+  setSelectedFilter,
+  clearCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todos.filter(todo => !todo.completed).length} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <button
```

Comment:
Use tag `a` for these links
```suggestion
        <a
```